### DISCUSSION TITLE
Payment Gateway

### DISCUSSION THREAD

**Discussion by kowalski21** [saleor/saleor#8408] [Q&A]
Can we please have a tutorial,on a payment gateway,
Its been a challenging process here

**Comment by karolkielecki** [comment]
@kowalski21 Hi! Could you please tell us where exactly it's challenging for you? 

If you don't know where to add a payment gateway on the storefront side, you can find more information here: https://spectrum.chat/saleor/saleor-storefront/modifying-the-storefront~c1955dbf-a421-4fb6-b99e-937dd2642b23

Have you seen our documentation regarding payment gateways? https://docs.saleor.io/docs/3.0/developer/extending/payment-gateways

**Reply by kowalski21** [reply]
@karolkielecki , Thanks for the response. I have reviewed the documentation, What I am trying to communicate is that 
if saleor could provide a step by step tutorial on say adding stripe payment in core graphql api
